assets/js/components/user-input/UserInputSelectOptions.js

```diff
@@ -124,9 +152,15 @@ export default function UserInputSelectOptions( { slug, options, max } ) {
 		);
 	} );
 
+	const handleKeyDown = useCallback( ( event ) => {
+		if ( event.keyCode === ENTER && other.trim().length > 0 && next && typeof next === 'function' ) {
+			next();
+		}
+	}, dependencies );
```

Comment:
The `next` variable also needs to be added to the dependencies list.

Reply:
👍